sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -343,4 +412,58 @@ trait StreamTest extends QueryTest with Timeouts {
     addCheck()
     testStream(ds)(actions: _*)
   }
+
+
+  object AwaitTerminationTester {
```

Comment:
This is used into two test suites - ContinuousQuerySuite and ContinuousQueryManagerSuite.